src/Epic/index.stories.tsx

```diff
@@ -55,6 +55,11 @@ export default {
             type: { name: 'string', required: false },
             description: 'Extra data to be associated with a reminder sign-up',
         },
+        showPrivacyText: {
+            name: 'showPrivacyText',
+            type: { name: 'string', required: false },
+            description: 'Set value to "true" to display the privacy text under the Reminder CTA',
```

Comment:
Is it worth mentioning the default behaviour here? I _think_ you said on epics the default is to show the privacy text?